index.html

```diff
@@ -66,17 +66,18 @@
       <div class="row">
         <div class="col-md-4">
           <h2>Heading</h2>
-          <p>Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Etiam porta sem malesuada magna mollis euismod. Donec sed odio dui. </p>
+          <p>Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Etiam porta sem malesuada magna mollis euismod. Donec sed odio dui.</p>
           <p><a class="btn btn-default" href="#" role="button">View details &raquo;</a></p>
         </div>
         <div class="col-md-4">
-          <h2>Heading</h2>
-          <p>Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Etiam porta sem malesuada magna mollis euismod. Donec sed odio dui. </p>
+          <h2>Eat</h2>
+          <p>Isn’t your <i>core</i> I imagine rain kept in musty nooks throw it looks will you stay silent <i>forever<i> I'm beyond the <i>rainbow</i> in the sky focused on anything one bite at a time.</p>
```

Comment:
` <p>Isn’t your <i>core</i> I imagine rain kept in musty nooks throw it looks will you stay silent <i>forever<i> I'm beyond the <i>rainbow</i> in the sky focused on anything one bite at a time.</p>`

For line 74, missing a / in the closing tag after the word forever.